src/Command/RecipesCommand.php

```diff
@@ -354,8 +355,9 @@ private function findRecipeCommitDataFromTreeRef(string $package, string $repo,
 
     private function requestGitHubApi(string $path)
     {
-        $contents = $this->downloader->getContents('api.github.com', $path, false);
+        /** @var Response $response */
```

Comment:
Do we need that ? Isn't it already covered by the composer phpdoc ?

Reply:
Reverted, not necessary.
Not the method is not documented.

Reply:
if the phpdoc is missing in composer, I suggest updating composer instead :smile: